hasher-matcher-actioner/terraform/.terraform.lock.hcl

```diff
@@ -43,6 +43,7 @@ provider "registry.terraform.io/hashicorp/local" {
 provider "registry.terraform.io/hashicorp/null" {
   version = "3.1.0"
   hashes = [
+    "h1:SFT7X3zY18CLWjoH2GfQyapxsRv6GDKsy9cF1aRwncc=",
```

Comment:
Also not sure what this change is?

Reply:
This is an auto-generated file from terraform. I have no idea why it's edited. @schatten might know